src/lvl_script_commands_old.c

```diff
@@ -599,7 +599,7 @@ static void command_add_creature_to_pool(const char *crtr_name, long amount)
         SCRPTERRLOG("Unknown creature, '%s'", crtr_name);
         return;
     }
-    if ((amount < 0) || (amount >= CREATURES_COUNT))
+    if ((amount < 0) || (amount >= game.conf.rules.game.creatures_count))
```

Comment:
Rules can change mid map. So it would make sense to be able to add more creatures to the pool than are currently allowed on the map.